util/src/test/java/io/kubernetes/client/CopyTest.java

```diff
@@ -69,13 +70,14 @@ public void testUrl() throws IOException, ApiException, InterruptedException {
                     .withBody("{}")));
 
     try {
-      copy.copyFileFromPod(pod, "container", "/some/path/to/file");
+      InputStream inputStream = copy.copyFileFromPod(pod, "container", "/some/path/to/file");
+      // block until the connection is established
+      inputStream.read();
```

Comment:
consider adding a timeout to fail the test in case the connection could not be established

Reply:
Thanks for your reply, I suppose if the wiremock connection could not be established, it would throw an exception and fail.
Maybe we could get this one merged and consider adding timeout in another PR.